core/src/main/resources/schema/app/20160628142654_CLOUD-59100_remove_service_componenttype.sql

```diff
@@ -0,0 +1,7 @@
+-- // CLOUD-61094_remove_service_componenttype
+-- Migration SQL that makes the change goes here.
+
+DELETE FROM component WHERE componenttype='SERVICE';
```

Comment:
we should set null the stack on every component which have reference for the stack